### PR TITLE
Cleanup pleblist

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

Cleaned up pleblist password/secret generation, also cleaned up the pleblist password/secret checking
Removed youtube entry from the example config file

### CODE REVIEW

**Comment by alazymeme** [comment]
Yeah I have no idea how to fix this issue. Someone'll have to help me

**Comment by pajlada** [comment]
If you click the "Details" button on the test that failed, you will see what failed. I've gone ahead and made the relevant changes :)

File: pajbot/web/__init__.py [hunk 1]
```diff
@@ -64,27 +64,6 @@ def init(args):
...
-        with open(args.config, "w") as configfile:
-            config.write(configfile)
-
-    if "secret_key" not in config["web"]:
-        salt = generate_random_salt()
-        config.set("web", "secret_key", salt.decode("utf-8"))
-
-        with open(args.config, "w") as configfile:
-            config.write(configfile)
-
```
Comment (pajlada) [inline pajbot/web/__init__.py]: This part does not belong to the Pleblist system. This should be taken back to its original state :)

Comment (RAnders00) [reply pajbot/web/__init__.py]: this belongs to pleblist 100%
where does it get used??

**Pull request merged**